electron-app/src/server/websites/telegram/telegram.service.ts

```diff
@@ -597,6 +598,8 @@ export class Telegram extends Website {
       FormContent.getDescription(defaultPart.data.description, submissionPart.data.description),
     );
 
+    console.log();
```

Comment:
I'll merge this time and clean this up, but try and clean these up in the future.

Reply:
Ohh didn't noticed it, idk how